### PR TITLE
Truly randomized the random function

### PR DESCRIPTION
**Pull request by raysarebest**
The `Math.random` function used to return a pseudorandom number, because computers can't generate true randomness. This is now fixed (see source).

### CODE REVIEW

**Comment by zachlatta** [comment]
This is amazing. Two things before merge:
- Can you re-add the newline at the end of the file? (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)
- Can you add an example of `Math.random`'s new usage and expected result in the README (probably at the bottom of the existing examples)?

**Comment by raysarebest** [comment]
Oh yeah, I forgot about that stuff. Just a second

File: README.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Math.pow(10.5);
 //             \//(____)
 //     ------\     (__)
 //            `-----"
+
+Math.random;
```
Comment (zachlatta) [inline README.md]: Shouldn't this be `Math.random()`?

File: index.js [hunk 2]
```diff
@@ -127,3 +127,6 @@ Math.pow = function (n) {
 
   return str;
 };
+
+// Chosen by a fair dice roll
+Math.random = 7;
```
Comment (zachlatta) [inline index.js]: Should this be a function that returns 7?

**Comment by raysarebest** [comment]
Just thought that would maximize the builds that would be broken, but that's a fair argument. Just a second

File: index.js [hunk 3]
```diff
@@ -127,3 +127,8 @@ Math.pow = function (n) {
 
   return str;
 };
+
+Math.random = function(){
```
Comment (zachlatta) [inline index.js:131]: Nitpick: can you change this to `Math.random = function() {` (space after the `()` to match the formatting of the other functions)

Comment (zachlatta) [inline index.js:131]: Actually, I take that back. Merging and will fix in another commit.

**Pull request merged**